scripts/restore

```diff
@@ -0,0 +1,183 @@
+port_component=$(ynh_app_setting_get --app=$app --key=port_component)
+
+#=================================================
+# CHECK IF THE APP CAN BE RESTORED
+#=================================================
+ynh_script_progression --message="Validating restoration parameters..." --time --weight=1
+
+ynh_webpath_available --domain=$domain --path_url=$path_url \
+	|| ynh_die --message="Path not available: ${domain}${path_url}"
+test ! -d $final_path \
+	|| ynh_die --message="There is already a directory: $final_path "
+
+#=================================================
+# STANDARD RESTORATION STEPS
+#=================================================
+# RESTORE THE NGINX CONFIGURATION
+#=================================================
+
+ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.conf"
+
+#=================================================
+# RESTORE THE APP MAIN DIR
+#=================================================
+ynh_script_progression --message="Restoring the app main directory..." --time --weight=1
+
+ynh_restore_file --origin_path="$final_path"
+
+#=================================================
+# RECREATE THE DEDICATED USER
+#=================================================
+ynh_script_progression --message="Recreating the dedicated system user..." --time --weight=1
+
+# Create the dedicated user (if not existing)
+ynh_system_user_create --username=$app --home_dir=$final_path
+
+#=================================================
+# RESTORE USER RIGHTS
+#=================================================
+
+# Restore permissions on app files
+chown -R root: $final_path
+
+#=================================================
+# SPECIFIC RESTORATION
+#=================================================
+# REINSTALL DEPENDENCIES
+#=================================================
+ynh_script_progression --message="Reinstalling dependencies..." --time --weight=1
+
+# Define and install dependencies
+ynh_install_app_dependencies $pkg_dependencies
+
+ynh_install_nodejs --nodejs_version=10
+
+#=================================================
+# CONFIGURE FIREWALL
+#=================================================
+ynh_script_progression --message="Configuring firewall..." --time --weight=1
+
+# Open this port
+ynh_exec_warn_less yunohost firewall allow --no-upnp TCP $port
+ynh_exec_warn_less yunohost firewall allow --no-upnp UDP $port_videobridge
+ynh_exec_warn_less yunohost firewall allow --no-upnp TCP $port_component
+
+#=================================================
+# CONFIGURE METRONOME
+#=================================================
+ynh_script_progression --message="Configuring metronome..." --time --weight=1
+
+# Create additional domains
+if [[ $domain == *"domain.tld"* ]]; then
+	yunohost domain add auth.$domain
+	yunohost domain add conference.$domain
+	yunohost domain add jitsi-videobridge.$domain
+	yunohost domain add focus.$domain
+fi
+#yunohost domain cert-install auth.$domain
```

Comment:
Uncomment these lines?

Reply:
As explained in https://github.com/YunoHost-Apps/jitsi_ynh/pull/4#discussion_r305083163, I removed them